src/platform/orchestrator/src/MpiApi.cpp

```diff
@@ -559,32 +685,29 @@ void Server::Worker(Server& server)
             }
 
             Request request;
-            if (ReadRequestFromSocket(connfd, request))
+            Response response;
+            if (ReadRequest(connfd, request))
             {
-                Response response = HandleRequest(request);
-                std::string responseString = response.ToString();
-
-                if (write(connfd, responseString.c_str(), responseString.size()) < 0)
-                {
-                    OsConfigLogError(PlatformLog::Get(), "Failed to write response to socket");
-                }
-                else if (IsFullLoggingEnabled())
-                {
-                    OsConfigLogInfo(PlatformLog::Get(), "Sending response: %s %d (%d bytes)", server.m_addr.sun_path, connfd, (int)responseString.size());
-                }
+                response = HandleRequest(request);
             }
             else
             {
                 OsConfigLogError(PlatformLog::Get(), "Failed to read request from socket");
```

Comment:
>OsConfigLogError(PlatformLog::Get(), "Failed to read request from socket"); [](http://example.com/codeflow?start=16&length=75)

Remove this line, it just duplicates traces written by ReadRequest w/o adding anything new